rasa_core/policies/fallback.py

```diff
@@ -63,6 +63,17 @@ def train(self,
 
         pass
 
+    def should_fallback(self,
+                        nlu_confidence,  # type float
+                        last_action_name  # type: Text
+                        ):
+        # type: (...) -> bool
+        """It should predict fallback action only if
+        a. predicted NLU confidence is lower than ``nlu_threshold`` &&
+        b. last action is NOT fallback action
+        """
+        return nlu_confidence < self.nlu_threshold and last_action_name != self.fallback_action_name  # noqa
```

Comment:
I don't think we should put a `noqa` here.

Reply:
Got it. Removed #qa and broke up into 2 lines to comply PEP8